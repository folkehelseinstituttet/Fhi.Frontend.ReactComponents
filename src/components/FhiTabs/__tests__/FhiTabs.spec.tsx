import { MemoryRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Tabs from '../FhiTabs';

describe('FhiTabs', () => {
  test('snapshot test', () => {
    const { container } = render(
      <MemoryRouter
        initialEntries={['/']}
        future={{
          v7_relativeSplatPath: true,
          v7_startTransition: true,
        }}
      >
        <Tabs links={[{ title: 'Fane nummer 1', path: '/1' }, { title: 'Fane nummer 2', path: '/2' }]} />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
