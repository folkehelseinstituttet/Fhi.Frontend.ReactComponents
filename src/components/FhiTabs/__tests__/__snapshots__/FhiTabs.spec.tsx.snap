// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FhiTabs snapshot test 1`] = `
<div>
  <div
    class="mb-7 nav nav-tabs"
  >
    <div
      class="nav-item"
    >
      <a
        class="nav-link"
        href="/1"
      >
        Fane nummer 1
      </a>
    </div>
    <div
      class="nav-item"
    >
      <a
        class="nav-link"
        href="/2"
      >
        Fane nummer 2
      </a>
    </div>
  </div>
</div>
`;
